framework/i18n/Formatter.php

```diff
@@ -1092,7 +1092,7 @@ public function asOrdinal($value)
      * If [[sizeFormatBase]] is 1024, [binary prefixes](http://en.wikipedia.org/wiki/Binary_prefix) (e.g. kibibyte/KiB, mebibyte/MiB, ...)
      * are used in the formatting result.
      *
-     * @param integer $value value in bytes to be formatted.
+     * @param string|integer|float $value value in bytes to be formatted.
```

Comment:
I wonder why [this](https://github.com/yiisoft/yii2/blob/master/tests/framework/i18n/FormatterNumberTest.php#L422) test passes:
```php
$this->assertSame("56.81 GB", $this->formatter->asShortSize((string)(28406984038 + 28406984038)));
```
Probably, it works only on 64-bit arch.

I think `(int)` may be changed to `(float)` in `formatSizeNumber`.